files/README.md

```diff
@@ -15,3 +15,28 @@ Then compile and run the service:
 go build
 ./files
 ```
+
+## Resources
+
+This example contains following two resources.
+
+* schema
+* swagger
+
+### Schema
+
+Schema is served from native file system so you have to deploy the binary with `public/schema` directory and keep their path relationship.
+
+### Swagger
+
+Swagger is served from file system of [https://github.com/elazarl/go-bindata-assetfs](github.com/elazarl/go-bindata-assetfs).
```

Comment:
The `swagger` resource also contains a single endpoint however it leverages [https://github.com/elazarl/go-bindata-assetfs](github.com/elazarl/go-bindata-assetfs) to serve assets embedded within the compiled binary.